src/main/java/skyler/Event.java

```diff
@@ -0,0 +1,45 @@
+package skyler;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that starts at a specific time
+ */
+public class Event extends Task {
+
+    protected LocalDateTime at;
```

Comment:
Maybe this variable name could be changed to make its purpose more clear without needing to see the context?